src/main/kotlin/vannak/tech/BookStockManagement/domain/models/Book.kt

```diff
@@ -0,0 +1,90 @@
+package vannak.tech.BookStockManagement.domain.models
+
+import vannak.tech.BookStockManagement.api.DTOs.BookDTO
+import vannak.tech.BookStockManagement.api.DTOs.CreateBookDTO
+import vannak.tech.BookStockManagement.api.DTOs.UpdateBookDTO
+import java.time.LocalDateTime
+import javax.persistence.*
+
+@Entity
+data class Book (
+        @Id
+        @GeneratedValue(strategy = GenerationType.AUTO)
+        @Column(name = "id")
+        var id:Long=0,
+        @Column(name = "title")
+        var title:String?,
+        @Column(name = "isbn")
+        var isbn:String?,
+        @Column(name = "author")
+        var author:String?,
+        @Column(name = "publisher")
+        var publisher:String?,
+        @Column(name = "edition")
+        var edition:Int?,
+        @Column(name = "price")
+        var price:Float?,
+        @Column(name = "quantity")
+        var quantity:Int?,
+        @Column(name = "status")
+        var status:String?,
+        @Column(name = "publish_year")
+        var publishYear:Int?,
+        @Column(name = "created_at")
+        var createdAt:LocalDateTime = LocalDateTime.now(),
+        @Column(name = "updated_at")
+        var updatedAt:LocalDateTime = LocalDateTime.now()
+) {
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
Please explain me. Why you use fetch in ManyToOne
You can chat to me in telegram.